crates/pallets/starknet/src/blockifier_state_adapter.rs

```diff
@@ -32,22 +33,15 @@ impl<T> StateChanges for BlockifierStateAdapter<T>
 where
     T: Config,
 {
-    fn count_state_changes(&self) -> (usize, usize, usize, usize) {
+    fn count_state_changes(
+        &self,
+    ) -> (UpdatedContractsCount, UpdatedStorageVarsCount, DeclaredClassesCount, DeclaredCompiledClassesCount) {
```

Comment:
That's a bit weird. Don't you want to just put them all in a single struct `StateChangesCount` with 4 well named fields?

Reply:
can do indeed yeah